test/tests/mesh/mesh_generation/tests

```diff
@@ -30,95 +30,117 @@
-    design = 'mesh/AnnularMesh.md'
-    issues = '#8432'
-  [../]
-
-  [./annular_except5]
-    type = RunException
-    input = annular_except5.i
-    expect_err = 'AnnularMesh: nt must be greater than \(tmax - tmin\) / Pi in order to avoid inverted elements'
-    requirement = "MOOSE's annular mesh generator shall throw an error if the number of elements in the angular direction is too small."
-    design = 'mesh/AnnularMesh.md'
-    issues = '#8432'
-  [../]
+    [./annular_except5]
+      type = RunException
+      input = annular_except5.i
+      expect_err = 'nt must be greater than \(dmax - dmin\) / 180 in order to avoid inverted elements'
+      detail = 'shall throw an error if the number of elements in the angular direction is too small.'
+    [../]
+    [./annular_except6]
+      type = RunException
+      input = annular_except6.i
+      expect_err = 'quad_subdomain_id must not equal tri_subdomain_id'
+      detail = 'shall throw an error if the quadrilateral and triangular subdomain ids are the same.'
+    [../]
 
-  [./annular_except6]
-    type = RunException
-    input = annular_except6.i
-    expect_err = 'AnnularMesh: quad_subdomain_id must not equal tri_subdomain_id'
-    requirement = "MOOSE's annular mesh generator shall throw an error if the quadrilateral and triangular subdomain ids are the same."
-    design = 'mesh/AnnularMesh.md'
-    issues = '#8432'
-  [../]
+    [./annulus]
+      type = Exodiff
+      input = annulus.i
+      exodiff = 'annulus_out.e'
+      detail = 'shall create an annulus using quad elements.'
+    [../]
 
-  [./annulus]
-    type = Exodiff
-    input = annulus.i
-    exodiff = 'annulus_out.e'
-    requirement = 'MOOSE shall have an algorithm for quad meshing an annulus.'
-    design = 'mesh/AnnularMesh.md'
-    issues = '#8432'
-  [../]
+    [./annulus_sector]
+      type = Exodiff
+      input = annulus_sector.i
+      exodiff = 'annulus_sector_out.e'
+      detail = 'shall create an annulus sector using quad elements.'
+    [../]
+    [./disc]
+      type = Exodiff
+      input = disc.i
+      exodiff = 'disc_out.e'
+      detail = 'shall create a circular disk using quad elements.'
+    [../]
 
-  [./annulus_sector]
-    type = Exodiff
-    input = annulus_sector.i
-    exodiff = 'annulus_sector_out.e'
-    requirement = 'MOOSE shall have an algorithm for quad meshing an annular sector.'
-    design = 'mesh/AnnularMesh.md'
-    issues = '#8432'
-  [../]
+    [./disc_sector]
+      type = Exodiff
+      input = disc_sector.i
+      exodiff = 'disc_sector_out.e'
+      detail = 'shall create a circular disk sector using quad elements.'
+    [../]
+  []
 
-  [./disc]
-    type = Exodiff
-    input = disc.i
-    exodiff = 'disc_out.e'
-    requirement = 'MOOSE shall have an algorithm for quad meshing a circular region, including the origin.'
-    design = 'mesh/AnnularMesh.md'
+  [./annular_deprecated]
     issues = '#8432'
```

Comment:
```suggestion
  deprecated = true
```

Reply:
Give this a try, if I recall the "deprecated" flag is applied in the same spot the "requirement" would be defined.

Reply:
got it, let's see how it does.

Reply:
Yup that did it! Thanks @aeslaughter